mono/metadata/w32handle.c

```diff
@@ -343,7 +384,7 @@ static guint32 mono_w32handle_new_internal (MonoW32HandleType type,
 					  gpointer handle_specific)
 {
 	guint32 i, k, count;
-	static guint32 last = 0;
+	guint32 last = 0;
```

Comment:
That a very weird change as in this case it will always try to allocate a handle from 0, and it won't resume from the last returned one.